File: cosmos-master/code/mathematical-algorithms/count_trailing_zeroes/Count_trailing_zeroes.scala
//Part of Cosmos by OpenGenus Foundation
object Count_trailing_zeroes {
        def trailingZeroes(n : Int) : Int = {
                var result :Int = 0;
                var i :Int = 5;

                while(i<n){
                        result += n/i;
                        i*=5;
                }

                return result;
        }

        def main(args: Array[String]){
                var n :Int = 13;
                //to test
                //print("Enter number:\n")
                //n = (Console.readLine).toInt;
                println("Trailing zeroes in " + n + "! : " + trailingZeroes(n)+'\n');
        }
}

File: cosmos-master/code/mathematical-algorithms/count_trailing_zeroes/count_trailing_zeroes.c
#include <stdio.h>
// Part of Cosmos by OpenGenus Foundation
int main()
{
	int t;
	scanf("%d",&t);
	while(t--)
	{
	int num,x=5,count=0;
	scanf("%d",&num);
	while(x<=num)
	{
		count=count+(num/x);
		x=x*5;
	}
	printf("%d\n",count);
}
	
}
